extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -3,49 +3,37 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import * as azdata from 'azdata';
-import { SqlMigrationImpactedObjectInfo } from '../../../../mssql/src/mssql';
-import { MigrationStateModel } from '../../models/stateMachine';
-import { Issues } from './assessmentResultsDialog';
-import { AssessmentDialogComponent } from './model/assessmentDialogComponent';
-
-type DbIssues = {
-	name: string,
-	issues: Issues[]
-};
-export class SqlDatabaseTree extends AssessmentDialogComponent {
-
-	public static excludeDbs: Array<string> = ['master', 'tempdb', 'msdb', 'model'];
-	private _model!: MigrationStateModel;
-	private instanceTable!: azdata.ComponentBuilder<azdata.DeclarativeTableComponent, azdata.DeclarativeTableProperties>;
-	private databaseTable!: azdata.ComponentBuilder<azdata.DeclarativeTableComponent, azdata.DeclarativeTableProperties>;
-	private _assessmentResultsTable!: azdata.ComponentBuilder<azdata.DeclarativeTableComponent, azdata.DeclarativeTableProperties>;
-	private _impactedObjectsTable!: azdata.ComponentBuilder<azdata.DeclarativeTableComponent, azdata.DeclarativeTableProperties>;
-	private _assessmentData: Map<string, Issues[]>;
+import { SqlMigrationAssessmentResultItem, SqlMigrationImpactedObjectInfo } from '../../../../mssql/src/mssql';
+import { MigrationStateModel, MigrationTargetType } from '../../models/stateMachine';
+export class SqlDatabaseTree {
+
+	private _instanceTable!: azdata.DeclarativeTableComponent;
+	private _databaseTable!: azdata.DeclarativeTableComponent;
+	private _assessmentResultsTable!: azdata.DeclarativeTableComponent;
+
+	private _impactedObjectsTable!: azdata.DeclarativeTableComponent;
 
 	private _recommendation!: azdata.TextComponent;
 	private _dbName!: azdata.TextComponent;
 	private _recommendationText!: azdata.TextComponent;
 	private _descriptionText!: azdata.TextComponent;
-	private _issues!: Issues;
 	private _impactedObjects!: SqlMigrationImpactedObjectInfo[];
 	private _objectDetailsType!: azdata.TextComponent;
 	private _objectDetailsName!: azdata.TextComponent;
 	private _objectDetailsSample!: azdata.TextComponent;
-	private _moreInfo!: azdata.TextComponent;
-	private _assessmentType!: string;
+	private _moreInfo!: azdata.HyperlinkComponent;
 	private _assessmentTitle!: azdata.TextComponent;
 
-	constructor(model: MigrationStateModel, assessmentData: Map<string, Issues[]>, assessmentType: string) {
-		super();
-		this._assessmentData = assessmentData;
-		this._model = model;
-		this._assessmentType = assessmentType;
-		if (this._assessmentType === 'vm') {
-			this._assessmentData.clear();
-		}
+	private _activeIssues!: SqlMigrationAssessmentResultItem[];
+	private _selectedIssue!: SqlMigrationAssessmentResultItem;
+
+	constructor(
+		private _model: MigrationStateModel,
+		private _targetTypes: string
```

Comment:
Should this just be `_targetType`? Also I would strongly suggest making an actual type value (can just be a string literal union) to ensure that the value is always what you expect.